CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+allow it to be rolled out more quickly, with some of the more complicated aspects
+(DReps and constitutional changes) rolled out in a second stage. This will also allow time
+for fuller evaluation of/consultation on incentives and other significant  issues, while establishing the basis for
+governance in Voltaire.
+Below is a high-level summary of an implementation plan for (very roughly) half of the plan.
+
+1. Add a subset of (important) governance actions to the transaction body (while deprecating protocol parameter updates and MIR certificates).
+  The following governance actions will be supported initially:
+  * protocol parameters
+  * hard forks
+  * withdrawals from the treasury
+  * Constitutional Committee changes
+2. Add votes to the transaction body, but disallow DRep votes.
+3. The first version of the new `RATIFY` rule will only count Constitutional Committee votes, plus SPO votes for hard forks (similarly to the previously defined, and now obsoleted, `CIP-47`).
+
+The remainder of this proposal will be implemented in the second stage.
+
+### Piggybacking on stake pool stake distribution
+
+The Cardano protocol is based on a Proof-of-Stake consensus mechanism, so using a stake-based governance approach is sensible. However, there are many ways by which one could define how to record the stake distribution between participants. As a reminder, network addresses can currently contain two sets of credentials: one to identify who can unlock funds at an address (a.k.a. payment credentials) and one that can be delegated to a stake pool (a.k.a. delegation credentials).
+
+Rather than defining a third set of credentials, we instead propose to re-use the existing delegation credentials, using a new on-chain certificate to determine the governance stake distribution. This implies that the DReps can (and likely will) differ from SPOs, so creating balance. On the flip side, it means that the governance stake distribution suffers from the same shortcomings as that for block production: for example, wallet software providers must support multi-delegation schemes and must facilitate the partitioning of stake into sub-accounts should an Ada holder desire to delegate to multiple DReps.
+
+However, this choice also limits the implementation effort for wallet providers down the line and minimizes the effort needed for end-users to participate in the governance protocol. The latter is a sufficiently significant concern to justify the decision. By piggybacking on the existing structure, we keep the system familiar to users and reasonably easy to set up in order to maximize the chance of success and the participation rate in the governance framework.
+
+### Separation of Hard Fork Initiation from Standard Protocol Parameter Changes
+
+In contrast to other protocol parameter updates, hard forks (or, currently, changes to the protocol's major version) require much more attention.
```

Comment:
are you suggesting that we rely on the implicit power, and not make the vote explicit?

Reply:
No. I guess not. I think there is value to the action, since there is a difference between initiation and ratification of HFs, but what I am saying is that this action _cannot_ be enacted without the SPOs, so the only vote that could be taken, in practice, is `no/blank`, because even a `yes` vote, might be a `no` vote, should the SPOs decide so.

But, arguing with myself here, I think I am misguided in this point, and there is no real gain in not having it here, while leaving it does have value, since it shows intent, and many other nice things such as consensus etc can be derived from it.

So I think I retract my previous thought.